FirstProject/Program.cs

```diff
@@ -1 +1,40 @@
-﻿
\ No newline at end of file
+﻿using FirstProject;
+using System.ComponentModel.Design;
+
+Console.WriteLine("FirstProject");
+
+
+    BasedCalculator basedcalculator = new BasedCalculator();
+Console.WriteLine("Kalkulator podstawowy");
+Console.WriteLine("Dodawanie - wybierz 1");
+Console.WriteLine("Odejmowanie - wybierz 2");
+Console.WriteLine("Mnożenie - wybierz 3");
+Console.WriteLine("Dzielenie - wybierz 4");
+int action = Convert.ToInt32(Console.ReadLine());
+    Console.WriteLine("Podaj liczby");
+    int x = Convert.ToInt32(Console.ReadLine());
+    int y = Convert.ToInt32(Console.ReadLine());
+if (action == 1)
+{
+    Console.WriteLine("Dodawanie");
+    int sum = basedcalculator.Addiction(x, y);
```

Comment:
myślę że w tym przypadku nie trzeba tworzyć zmiennej dodatkowej tylko od razu wynik dać do WriteLinea. Gdyby były 3 metody to już by można było pomyślec nad czytelnością ale jak to tylko jest metoda w metodzie to zaoszczędzimy linię jedną w każdym z przypadków.